src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalPrice = days * 40;
+
+  if (days < 3) {
```

Comment:
Create separate values for all numbers, don't harcode
5 point in [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)